packages/sparkify/src/executors/ensure/executor.ts

```diff
@@ -0,0 +1,46 @@
+import * as fs from 'fs'
+
+export interface EnsureExecutorOptions {
+  organisation: string
+  package: string
+}
+
+export default async function ensureExecutor(
+  options: EnsureExecutorOptions
+): Promise<{ success: boolean }> {
+  process.stdout.write(
+    `Ensuring @${options.organisation}/${options.package} package...`
+  )
+
+  let success = false
+  let packageJson = {} as any
```

Comment:
This `any` throws a lint warning. You can solve it by adding a type for `package.json`, something like:

```ts
// Add this outside the executor
export type PackageJson = {
  name?: string
  version?: string
  publishConfig?: {
    directory?: string
  }
}
```

```suggestion
  let packageJson: PackageJson = {}
```